sdk/communication/Azure.Communication.Chat/CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 # Release History
 
 ## 1.1.0-beta.1 (Unreleased)
+### Added
+- Support for metadata in message. 
```

Comment:
Format should be "Added support for metadata in message." "Added new overloads for sendmessage and updatemessage operations with options : SendChatMessageOptions and UpdateChatMessageOptions"